Python/optimizer.c

```diff
@@ -995,16 +997,43 @@ count_exits(_PyUOpInstruction *buffer, int length)
 {
     int exit_count = 0;
     for (int i = 0; i < length; i++) {
-        int opcode = buffer[i].opcode;
-        if (opcode == _EXIT_TRACE) {
+        uint16_t base_opcode = _PyUop_Uncached[buffer[i].opcode];
+        if (base_opcode == _EXIT_TRACE) {
             exit_count++;
         }
     }
     return exit_count;
 }
 
+static int
+get_exit_depth(_PyUOpInstruction *inst)
```

Comment:
Can you write a short snippet on what this does? It's rather confusing otherwise. IIUC, it finds what is the number of "used" registers on exit right?

Reply:
Where "exit" means a side exit (EXIT_IF or DEOPT_IF) not the end of the uop.